docs/datamodel/scalars/numeric.rst

```diff
@@ -103,6 +103,16 @@ from :eql:type:`str` and :eql:type:`json`.
         db> SELECT 42n IS bigint;
         {true}
 
+    To represent really big integers it is possible to even use the
+    exponent notation (e.g. ``1e20`` instead of
+    ``100000000000000000000``) as long as the exponent is positive and
+    there is not dot in the mantissa.
```

Comment:
I think "mantissa" is what follows the dot, no?

Reply:
I had only seen it used to mean that as well, but apparently it can mean either: https://en.wikipedia.org/wiki/Scientific_notation

Reply:
I guess I'll reword it as "no dot anywhere" then, to avoid confusion.